common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -563,7 +563,8 @@
                             if ((stringValue.indexOf('{{') !== -1) && (stringValue.indexOf('}}') !== -1)) {
                                 stringValue = stringValue.replace(/{{[\s\S]*?}}/g, '').trim();
                             }
-                            if (stringValue.match(/[a-z]/i)) {
+                            // allow for "e" for scientific notation, otherwise, exclude letters
+                            if (stringValue.match(/[a-df-z]/i)) {
```

Comment:
This change would ideally be accompanied by a test in this file:
`common/lib/xmodule/xmodule/js/spec/problem/edit_spec.js`
as this situation arose due to the absence of a proper unit test.

Reply:
I added the test case. Thanks for finding the right file!